projects/slap_manipulation/scripts/eval_slap_agent.py

```diff
@@ -0,0 +1,58 @@
+import click
+import numpy as np
+import rospy
+import trimesh.transformations as tra
+from slap_manipulation.agents.slap_agent import SLAPAgent
+from slap_manipulation.env.general_language_env import GeneralLanguageEnv
+
+from home_robot.core.interfaces import ContinuousEndEffectorAction
+from home_robot_hw.env.stretch_pick_and_place_env import load_config
+
+
+@click.command()
+@click.option("--task-id", default=-1)
+def main(task_id, **kwargs):
+    config = load_config(
+        visualize=True,
+        config_path="projects/slap_manipulation/configs/language_agent.yaml",
+        **kwargs
+    )
+    rospy.init_node("eval_slap")
+    agent = SLAPAgent(config, task_id=task_id)
+    agent.load_models()
+
+    env = GeneralLanguageEnv(
+        config=config,
+        segmentation_method="detic",
+    )
+
+    env.reset()
+    agent.reset()
+
+    goal_info = agent.get_goal_info()
```

Comment:
this seems like mostly duplicated code from the peract script? how are they different, should they be combined?

Reply:
What did you mean here @cpaxton ? `eval_peract_agent.py` is substantially different from `eval_slap_agent.py`. We could add another wrapper over current PeractAgent and consolidate, but I think that's out of scope of this PR. I can add an issue for repo owners to use in the future though.

Reply:
I don't understand why they should be different? they're both taking in the same observations and returning the same actions

Reply:
Hmmm I can move the logic for handling multiple waypoints inside PerActAgent which would make the outer loops the same. I will test tomorrow.

Reply:
SLAP evaluation has been merged with GeneralLangAgent logic.